src/main_updated.py

```diff
@@ -0,0 +1,165 @@
+    elif len(tuples) == 0 and username != username2:
+        return username2
+
+#function to check that all passwords meet the required criteria
+def check_pw(password):
+    Capital = False
+    digit = False
+    alnum = False
+    while Capital == False and digit == False and alnum == False:
+        for i in password:
+          if i.isupper():
+             Capital = True
+          if i.isdigit():
+             digit = True
+          if not i.isalnum():
+              alnum = True
+        if Capital == True and digit == True and alnum == True:
+            if len(password) < 8 or len(password) > 12:
+                print("Password must be at least 8 and no more than 12 characters in length.")
+                password = getpass()
+                check_pw(password)
+        elif Capital != True:
+            print("Password must contain at least one capital letter")
+            password = getpass()
+            check_pw(password)
+        elif digit != True:
+            print("Password must contain at least one digit")
+            password = getpass()
+            check_pw(password)
+        elif alnum != True:
+            print("Password must contain at least one alphanumeric symbol")
+            password = getpass()
+            check_pw(password)
+
+#function to check the amount of accounts that have been created
+def number_rows():
+    query = """SELECT * FROM Username"""
+    c.execute(query)
+    conn.commit()
+    rows = len(c.fetchall())
+    #print("The number of rows is ", rows)
+    return rows
+    
+def find_in_db(fn,ln):
+    
+    for row in c.execute("""SELECT * FROM Username"""):
+        if fn == row[2] and ln == row[3]:
+            print("They are a part of the InCollege system")
+            return
+    print("They are not yet a part of the InCollege system yet")
+
+#function to login into the program for user that have already created an account
+def login_attempt(username, password):
+    query = """SELECT * FROM Username WHERE username = ? AND password = ?;"""
+    data = (username, password)
+    c.execute(query, data)
+    conn.commit()
+    tuple = c.fetchall()
+   
+    return len(tuple) != 0
+
+#CHOICE IS A CHAR THAT HELPS NAVIGATE THROUGH THE PROGRAM MENU
+def main():
+    choice = '?'
+
+    while choice != 'q':
+        print("\n")
+        print("          MENU     ")
+        print("n - Create new account")
+        print("l - Login")
+        print("q - Quit")
+        print("f - find a friend")
+        print("s - success story")
+        print("\n")
+
+        question = input("Please make a choice from the menu: ")
+        
+        choice = question
+        
+        if choice=='f':
+            fn=input("enter first name:" )
+            ln=input("enter last name:" )
+            find_in_db(fn,ln)
+            rn=input("please enter to the previous page")
+        if choice=='s':
+            print("video is now playing")
+            rn=input("press enter to the previous page")
```

Comment:
I believe the success story is to be displayed by default before the program menu (you can just write a made-up story that you can call before the menu choices from my understanding), and the option to play a video would then replace the "s- success story" selection. I have attached below the section from the description which describes this: 

"A user who has not signed in will see a college student success story displayed on the home screen – they used InCollege to get a job. The college student success story will be made up by you. The user is provided with an option that will lead to a video that will explain why they would want to join InCollege. When the user selects to play a video, a message that says "Video is now playing" will be displayed. "